### ISSUE TITLE
[REVIEW]: feign: a Python package to estimate geometric efficiency in passive gamma spectroscopy measurements of nuclear fuel

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#1650]
**Submitting author:** @ezsolti (<a href="http://orcid.org/0000-0003-2339-4340">Zsolt Elter</a>)
**Repository:** <a href="https://github.com/ezsolti/feign/" target ="_blank">https://github.com/ezsolti/feign/</a>
**Version:** v1.0.0
**Editor:** @katyhuff
**Reviewer:** @kellyrowland, @sskutnik
**Archive:** <a href="https://doi.org/10.5281/zenodo.3480082" target="_blank">10.5281/zenodo.3480082</a>

## Status

[![status](https://joss.theoj.org/papers/d6d0fa620abbd1777d73190b1038602a/status.svg)](https://joss.theoj.org/papers/d6d0fa620abbd1777d73190b1038602a)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/d6d0fa620abbd1777d73190b1038602a"><img src="https://joss.theoj.org/papers/d6d0fa620abbd1777d73190b1038602a/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/d6d0fa620abbd1777d73190b1038602a/status.svg)](https://joss.theoj.org/papers/d6d0fa620abbd1777d73190b1038602a)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/ezsolti/feign/" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@kellyrowland & @sskutnik, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @katyhuff know.

✨ **Please try and complete your review in the next two weeks** ✨ 

### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ezsolti/feign/">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Version:** Does the release version given match the GitHub release (v1.0.0)?
- [x] **Authorship:** Has the submitting author (@ezsolti) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?

### Conflict of interest

- [x] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ezsolti/feign/">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Version:** Does the release version given match the GitHub release (v1.0.0)?
- [x] **Authorship:** Has the submitting author (@ezsolti) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @kellyrowland, @sskutnik it looks like you're currently assigned to review this paper :tada:.

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
Hello @kellyrowland, @sskutnik. Thank you for taking on the review. Please consider that I have committed most of the work as "zsolt" and realized this only after like 70 commits, when it was too late to change. Next time I will be more careful:)

**Comment by kellyrowland** [comment]
Hi @ezsolti , it looks like JOSS requires a code release with a version number matching the one listed here, but there appear to be not releases on the source code repo.

**Comment by katyhuff** [comment]
@ezsolti : This issue regarding authorship can be helped with some rewriting of history, if you like...  https://help.github.com/en/articles/changing-author-info

**Comment by katyhuff** [comment]
@kellyrowland @ezsolti : Since this will be the "v.1.0.0" release, it is not uncommon for authors to submit to JOSS before the release, and then, after making the edits requested by the reviewers (but not other major edits), make the v.1.0.0 release (since a release must be created for zenodo anyway, which is the last step of the JOSS publication.) 

For now, let's assume this review process will end with a DOI that points to a v.1.0.0 release, but that all reviews are conducted at the most recent commit (at the time of submission: 61978385bd1b950ea66a9704572484dc20b93b34 ). @ezsolti This will require that, until acceptance and until after the release, you only make edits to your master branch that were requested by these two JOSS authors. Continuing development can happen in parallel of course, on an experimental branch -- but should not be included in the release, as it will not be included in the review.

**Comment by kellyrowland** [comment]
Thanks for the clarification @katyhuff !

**Comment by ezsolti** [comment]
Yes, thank you for the clarification @katyhuff. I am not supposed to change the code (which i havent done), or anything in the repo, like minor typos in the the readme (which i have done, but can go back to the original commit)?

**Comment by ezsolti** [comment]
Also, I have managed to rewrite the history, thanks for the help!

**Comment by katyhuff** [comment]
@ezsolti : very minor changes are fine, as are changes related to satisfying the recommendations of the reviewers. It's just that, for cleanliness of the meaning of the review process, major changes shouldn't be incorporated mid-review (unless called for by the review process itself). Don't worry, we try to keep it really quick, and you can always work in another branch to keep your master branch clean.

**Comment by katyhuff** [comment]
Thanks @kellyrowland ! 

@sskutnik : I know it's a busy time. Please let us know if you expect this review to take more than a week or so to complete.

**Comment by sskutnik** [comment]
I've made a few minor comments re: the paper on ezsolti/feign/issues/7

In general, I'd recommend minor revisions, in part for clarity and also to clear up the theory of the efficiency calculation; also to update the references such that they render with fully-locatable citations. Otherwise, I think this looks fine.

**Comment by katyhuff** [comment]
Thanks @sskutnik ! Excellent first review!

@ezsolti : I agree with Prof. Skutnik regarding clarity. When you feel you've handled these comments (in  ezsolti/feign/issues/7 mostly) please let me know (by @-mentioning me in this issue.) and we'll move forward.

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
@katyhuff : I have tried to address the comments of Prof. Skutnik, and as you see I have also tried to render a new paper based on some comments which did not require further "iterations" (I hope that this update was not against the rules). Unfortunately, one reference still renders strangely. Let's see whether @sskutnik would advise further changes based on the discussion in ezsolti/feign/issues/7 .

**Comment by sskutnik** [comment]
I think I'm happy with the proposed changes; I'd leave it to @ezsolti whether to include the figure provided in https://github.com/ezsolti/feign/issues/7

Otherwise, I think all of my concerns have been addressed and I would happily recommend this to move forward for publication.

**Comment by ezsolti** [comment]
@sskutnik : Thinking it over again, i might be on the opinion that the figure in ezsolti/feign#7 or a similar one for more pins should rather be included in the documentation when reasoning why using the random source case is superior to the center case in certain setups. Feels like it would overcomplicate the paper which has already reached the recommended upper limit for length.

**Comment by sskutnik** [comment]
@ezsolti This seems like a reasonable conclusion. I think the documentation is a reasonable alternative.

**Comment by katyhuff** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
Hello @katyhuff : sorry, i have been on parental leave for the last 2 weeks, and have overlooked the mail from this thread.

**Comment by katyhuff** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.13182/NT11-135 is OK
- 10.1109/NSSMIC.2017.8533017 is OK

MISSING DOIs

- None

INVALID DOIs

- https://doi.org/10.1016/j.nima.2016.07.032 is INVALID because of 'https://doi.org/' prefix
```

**Comment by katyhuff** [comment]
@ezsolti I have reviewed the code and the paper. I have some comments on the paper which can be found in the attached pdf: 
[joss-feign.pdf](https://github.com/openjournals/joss-reviews/files/3638206/joss-feign.pdf)



Primarily: 
- [ ]  In your references, the DOI https://doi.org/10.1016/j.nima.2016.07.032 is INVALID because of 'https://doi.org/' prefix
- [ ] The statement of need and the description of the intended usership of the work could be much more clear and should be stated earlier in the paper.
- [ ] There are a handful of grammatical errors (see pdf)
- [ ] The side-by-side figures would be improved by matching legend maxima so that the colors have the same meaning in adjacent images.
- [ ] The second paragraph is very long and quite heavy in jargon. The average JOSS reader will find it impenetrable. Consider more clear, jargon-free language.
- [ ] The paper should cite software on which feign relies (numpy, matplotlib)

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
@katyhuff Thank you for your comments, I've managed to address most of them:

- I have removed the prefix
- I have clearly included the intended usership in the 2nd section.
- I modified the grammar mistakes. There was one mark in the review on the 2nd page ("if the background is removed"), what I was not sure about whether it was deliberate.
- I have tried before with the matching legend maxima, unfortunately it does not look nice, since there is an order of magnitude difference. Logarithmic scale also doesnt improve it visually. Alternatively, I have normalized with the sum of the pinwise contributions, and plotted a relative pinwise contribution in percentage. This one looks informative even when the range is the same on the colorbar. Please, let me know whether you agree with this change.
 - Regarding the second paragraph being heavy, I have a hard time here, since I grew to like that part, but I also understand if you feel it problematic for JOSS readers. I have to think how to modify it in a way that I don't feel I'm loosing information. If you mean specifically the F5 and F8 referencing, I could remove that though I have added it with the intention to serve as an extra clarification for possible nuclear readers. 
- Similarly, I agree with you that it is a bit awkward that the feign section starts only after the rather lengthy summary. I have seen this format (ie. summary section, then package section) in several JOSS papers, and I liked it, but indeed, the intended usership gets a bit far from the beginning of the article. I will need some time to think about how to change it without making the paper worse:)
- I have included a sentence on what NumPy and Matplotlib is used for and added references. I'm not sure whether it smoothly fits there.

**Comment by katyhuff** [comment]
@ezsolti Thank you for working toward addressing my comments. 

Please let me know when you are able to:

- [ ] Reduce the jargon in the 2nd paragraph (indeed, throughout the paper, which is already unusually long for a JOSS paper). The F5 tally mention is not the only item that I find problematic from a jargon perspective. Please aim to make clear the high-level functionality and purpose of the software for a diverse, non-specialist audience.
- [ ] Make clear, much earlier and more succinctly, a Statement of Need that illustrates the research purpose of the software. (e.g.  what problems the software is designed to solve and who the target audience is).

Please note, these are not merely my suggestions, but are requirements at the core of our submission instructions, which can be found here:

https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain


![image](https://user-images.githubusercontent.com/393899/65655852-433aad80-dfe3-11e9-9ad4-ee080a149c57.png)

**Comment by ezsolti** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01650/joss.01650/10.21105.joss.01650.pdf)

**Comment by ezsolti** [comment]
Hello @katyhuff , after reading through couple of more JOSS articles, I saw that some authors directly include a Statement of Need sections, so I tried to go with that. I also dropped the jargon. Please let me know what do you think of the text now?

**Comment by katyhuff** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.13182/NT11-135 is OK
- 10.1109/NSSMIC.2017.8533017 is OK
- 10.1016/j.nima.2016.07.032 is OK
- 10.1109/MCSE.2011.37 is OK
- 10.1109/MCSE.2007.55 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by katyhuff** [comment]
Thank you @kellyrowland and @sskutnik for your reviews -- we couldn't do this without you.
Thank you @ezsolti for your submission and for engaging actively in the review process! I have looked over the paper, double-checked all the DOI links, and have conducted a high-level review of the code itself. Everything looks ship-shape to me. At this point, please double-check the paper yourself, review any lingering details in your code/readme/etc., and then make an archive of the reviewed software in Zenodo/figshare/other service. **Please be sure that the DOI metadata (title, authors, etc.) matches this JOSS submission.** If you need to update the version, that's fine, please just let me know if the version in the DOI is different from the version named in the submission. Once that's complete, please update this thread with the DOI of the archive, and I'll move forward with accepting the submission. Until then, now is your moment for final touchups!

**Comment by ezsolti** [comment]
Thank you @katyhuff !

The DOI for the archive: 10.5281/zenodo.3480082
The version is the same as in this submission!

**Comment by katyhuff** [comment]
@whedon set 10.5281/zenodo.3480082 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.3480082" target="_blank">10.5281/zenodo.3480082</a> is the archive.

**Comment by katyhuff** [comment]
@openjournals/joss-eics I believe this is ready for acceptance. Over to you!

**Comment by labarba** [comment]
@ezsolti — I'm the Associate Editor-in-Chief on rotation this week. I made a few edits on the paper via PR. Unfortunately the diff is not too helpful, because long paragraphs were written on the same source line. The top fix I wanted to enter was an in-text citation that didn't use the right [citation syntax](https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax ). I also removed a few unnecessary "Then" and changed "analyze" to US spelling, plus little things.

**Comment by ezsolti** [comment]
@labarba Thank you! I have merged your commit.

**Comment by labarba** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.13182/NT11-135 is OK
- 10.1109/NSSMIC.2017.8533017 is OK
- 10.1016/j.nima.2016.07.032 is OK
- 10.1109/MCSE.2011.37 is OK
- 10.1109/MCSE.2007.55 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1025

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1025, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by labarba** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1183361917152350208) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1026
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.01650](https://doi.org/10.21105/joss.01650)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? notify your editorial technical team...

**Comment by labarba** [comment]
Congratulations, @ezsolti, your JOSS paper is published! 🚀 

Huge thanks to our editor: @katyhuff, and reviewers: @kellyrowland, @sskutnik — your contributions to JOSS are greatly appreciated 🙏

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.01650/status.svg)](https://doi.org/10.21105/joss.01650)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.01650">
  <img src="https://joss.theoj.org/papers/10.21105/joss.01650/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.01650/status.svg
   :target: https://doi.org/10.21105/joss.01650
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.01650/status.svg)](https://doi.org/10.21105/joss.01650)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss
